.github/workflows/backend-ci-cd.yml

```
@@ -0,0 +1,79 @@
+        run: |
+          for service in user-service file-service sync-service gateway; do
+            cd $service
+            mvn verify -DskipUnitTests=true
+            cd ..
+          done
+      - name: Generate Final Coverage Report
+        working-directory: apps/backend
+        run: |
+          mvn verify -DskipTests=true
+      - name: SonarCloud Scan
+        run: |
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)